index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200&display=swap"
+    rel="stylesheet">
+</head>
+<body>
+  <section class="container">
+    <nav class="navbar d-flex justify-content-between">
+      <div class="logo hover">
+        My Logo
+      </div>
+      <div class="menu hover">
+        <img src="img/Union.png" alt="Menu button">
+      </div>
```

Comment:
- [ ] Good work including your burger menu here ✔️. However, please kindly make sure that you wrap your mobile menu using a `nav` semantic tag instead of a `div` tag to make your site more accessible 🤓.